azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/models/CallCompositeErrorCode.java

```diff
@@ -27,6 +27,11 @@ public final class CallCompositeErrorCode extends ExpandableStringEnum<CallCompo
      */
     public static final CallCompositeErrorCode TOKEN_EXPIRED = fromString("tokenExpired");
 
+    /***
```

Comment:
It is public API change. We may need review for this. Please discuss with Pavel.

Reply:
We (android & iOS) had plans for review but we got the response that we don't need to sit for review to add only an error code.